server/controllers/userController.js

```diff
@@ -0,0 +1,33 @@
+import models from '../models';
+
+const { User, Profile } = models;
+/**
+ * @description - Performs all auth function
+ */
+class UserController {
+  /**
+   * @param {object} req - req sent to server
+   * @param {object} res - response gotten from server
+   * @param {fn} next - csllback function
```

Comment:
A typo here: `csllback`should be `callback`

Reply:
Done, thanks.